packages/tools/kadena-cli/src/tx/commands/templates/templates.ts

```diff
@@ -50,6 +50,24 @@ networkId: "{{network:networkId}}"
 type: exec
 `;
 
+export const arbitraryCodeTemplate = (
+  code: string,
+  chainId: string,
+  network: string,
+): string => `
+code: |-
+  ${code}
+data:
+meta:
+  chainId: "${chainId}"
+  gasLimit: 2300
```

Comment:
I would make the gaslimit optional also. When sending large bits of code you might have to increase it.

Reply:
is it even possible to submit a transaction without gasLimit? shouldn't I just hardcode it at a very high value then?

Reply:
optional wasn't the right word, should have been configurable. we can keep this as a default but to option to increase the value should be there